camel/agents/chat_agent.py

```diff
@@ -191,6 +191,7 @@ def __init__(
         self.func_dict = {
             tool.get_function_name(): tool.func for tool in all_tools
```

Comment:
@Wendong-Fan should we add `__call__` in `FunctionTool` so we do not need to acess `FunctionTool.func` outside?
```suggestion
            tool.get_function_name(): tool for tool in all_tools
```